### PR TITLE
lecture3 slide 41 missprint fix

### PR DESCRIPTION
**Pull request by dmvolod**

### CODE REVIEW

File: lesson3.slide [hunk 1]
```diff
@@ -283,8 +283,8 @@ Outline:
 - *dynamic* *type*: type of assigned _value_
 - *dynamic* *value*: the actual value assigned
 - where's the problem?
-- interface type value is `nil` iff both dynamic value *and* dynamic type are `nil`
-- sometimes we get the impression that `nil != nil`
+- interface type value is `nil` if both dynamic value *and* dynamic type are `nil`
```
Comment (tisnik) [inline lesson3.slide:286]: `iff` is not a typo there, it means `if and only if`
https://en.wikipedia.org/wiki/If_and_only_if

File: lesson3.slide [hunk 2]
```diff
@@ -283,8 +283,8 @@ Outline:
...
+- sometimes we get the impression that `nil`!=`nil`
```
Comment (tisnik) [inline lesson3.slide:287]: looks ok, thank you very much!

@dmvolod are you taking this course?

File: lesson3.slide (hunk 1 above)
Comment (dmvolod) [reply lesson3.slide]: Sorry, @tisnik didn't know about this meaning in English

File: lesson3.slide (hunk 2 above)
Comment (dmvolod) [reply lesson3.slide:287]: Yes, @tisnik , very good content. Thank you.

Comment (tisnik) [reply lesson3.slide:287]: thank you. Good to know!

**Pull request merged**